packages/cmrjs/tests/cmr-utils/test-reconcilecmrmetadata.js

```diff
@@ -365,3 +365,30 @@ test('publishUMMGJSON2CMR calls ingestUMMGranule with ummgMetadata via valid CMR
   restoreCMR();
   sinon.restore();
 });
+
+test('publishUMMGJSON2CMR calls ingestUMMGranule and throws an error due to a revision ID version conflict', async (t) => {
```

Comment:
If I'm correct that our code doesn't need to check for revision ID conflicts, then I think this test is unnecessary

Reply:
Ah, I think you are correct. Reading the CMR docs, I believe CMR will throw the error.